scripts/convert_cogvideox_to_diffusers.py

```diff
@@ -89,6 +95,8 @@ def replace_up_keys_inplace(key: str, state_dict: Dict[str, Any]):
     "freqs_sin": remove_keys_inplace,
     "freqs_cos": remove_keys_inplace,
     "position_embedding": remove_keys_inplace,
+    # TODO  zRzRzRzRzRzRzR: really need to remove?
+    "pos_embedding": remove_keys_inplace,
```

Comment:
Note that CogVideoX-5b-I2V uses learned positional embeddings after the patch embedding as well as RoPE embeddings on QK.

TODO: As discussed with Yuxuan, we are able to generate videos without the learned positional embeddings but ideally this is needed even if it's role is minimal. This will limit multi-resolution or multi-frame generation as we can't dynamically generate the learned embeddings on-the-fly. Will work together on adding this